neo/Consensus/ConsensusService.cs

```diff
@@ -716,15 +584,6 @@ public static Props Props(IActorRef localNode, IActorRef taskManager, Store stor
             return Akka.Actor.Props.Create(() => new ConsensusService(localNode, taskManager, store, wallet)).WithMailbox("consensus-service-mailbox");
         }
 
-        private byte GetLastExpectedView(int validatorIndex)
```

Comment:
This was what gave it support for receiving `RecoveryMessage` without the `ChangeView` messages present, however, since `ChangeView` messages are now always present in the `RecoveryMessage` with this change, so it doesn't have to have extra code to handle this, with the trade-off being the additional time to verify the `ChangeView` witnesses in all cases now.